tests/test_ops/test_assign_score_withk.py

```diff
@@ -186,3 +186,110 @@ def test_paconv_assign_scores():
         points.grad.detach().cpu(), expected_points_grad, atol=1e-6)
     assert torch.allclose(
         centers.grad.detach().cpu(), expected_centers_grad, atol=1e-6)
+
+
+def test_paconv_assign_scores_cpu():
```

Comment:
You can share the test code with `pytest.mark.parametrize` like https://github.com/open-mmlab/mmcv/blob/1db3967e860569c1501af4bff6c0964dec402289/tests/test_ops/test_nms.py#L22